internal/repository/migrations/000001_create_application_input_claim_output_report_nodeconfig.up.sql

```diff
@@ -0,0 +1,88 @@
+-- (c) Cartesi and individual authors (see AUTHORS)
+-- SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+CREATE TYPE "ApplicationStatus" AS ENUM ('RUNNING', 'NOT RUNNING');
+
+CREATE TYPE "InputCompletionStatus" AS ENUM ('NONE', 'ACCEPTED', 'REJECTED', 'EXCEPTION', 'MACHINE_HALTED', 'CYCLE_LIMIT_EXCEEDED', 'TIME_LIMIT_EXCEEDED', 'PAYLOAD_LENGTH_LIMIT_EXCEEDED');
+
+CREATE TYPE "ClaimStatus" AS ENUM ('PENDING', 'SUBMITTED', 'FINALIZED');
+
+CREATE TYPE "DefaultBlock" AS ENUM ('FINALIZED', 'LATEST', 'PENDING', 'SAFE');
+
+CREATE TABLE "application"
+(
+    "id" SERIAL,
+    "contract_address" BYTEA NOT NULL,
+    "template_hash" BYTEA NOT NULL,
+    "snapshot_uri" VARCHAR(4096) NOT NULL,
+    "last_processed_block" NUMERIC(20,0) NOT NULL,
+    "status" "ApplicationStatus" NOT NULL,
+    "epoch_length" INT NOT NULL,
+    CONSTRAINT "application_pkey" PRIMARY KEY ("id"),
+    UNIQUE("contract_address")
+);
```

Comment:
Don't we need a `chainId`, I believe it's possible to have two applications with the same `contractAddress` but on different chains, right?

Reply:
Yes, that's possible but dealing with multiple chains is out of the current scope, so `chainId` is not necessary for now.